SparkPost/2web2ui
# SA-1344: Updates for date picker

### What Changed
 - Added hibana datepicker component (replaces DateSelector)
 - Added label to date selector
 - Fixed the precision

### How To Test
 - Go to Report Builder
 - Check that the date picker is working
 - Check that the precision is working
 - Confirm SA-1481 "Empty charts in report builder on initial load" has been fixed

### To Do
- [ ] Double check failing precisions (24H, 1 min and 15 min)(1 week, 1 hour)
